### PR TITLE
Add supportsFraction and intervalDecimal to SE example

### PR DESCRIPTION
**Pull request by andreas-junghanns**

### CODE REVIEW

**Comment by andreas-junghanns** [comment]
I think this example misses not just supportsFraction but also the required intervalDecimal.

**Review by nickbattle — APPROVED** [review]
I've applied these changes and re-checked with VDMCheck3.sh. It now gives the following:
```
nick@pollux:~/Digital Twins/fmi-standard/docs/examples> VDMCheck3.sh scheduled_execution.xml 
Checking XML
1028: 2.4.9 InitialUnknowns must include refs: {2, 4}
Errors found.
```
VRs 2 and 4 are the output clocks, so this may be an error in the checker - it looks like clocked variables should be excluded from initialUnknowns "outputs".

**Comment by nickbattle** [comment]
_> VRs 2 and 4 are the output clocks, so this may be an error in the checker_

Yes, it was excluding clocks from the test, not clocked variables. Will push the updated ZIP soon.

File: docs/examples/scheduled_execution.xml [hunk 1]
```diff
@@ -18,15 +18,15 @@
 		<!-- Clock variables -->
 		<!-- Periodic input clock -->
 		<Clock name="10msClock"      valueReference="5" causality="input"  intervalVariability="constant"
-		priority="1" intervalCounter="10" resolution="1000"/>
+		priority="1" supportsFraction="true" intervalDecimal="0.01" intervalCounter="10" resolution="1000"/>
```
Comment (MBlesken) [inline docs/examples/scheduled_execution.xml:21]: @andreas-junghanns
* isn't intervalCounter="10" resolution="1000" equivalent to intervalDecimal="0.01"? So why change it? ... why leave both options in the standard?
* supportsFraction="true" means that "...  fmi3GetXXXFraction and fmi3SetIntervalFraction are allowed to be called ..." - I don't see a need to call it for a constant clock. Do you?

Comment (andreas-junghanns) [reply docs/examples/scheduled_execution.xml:21]: * yes, it is the same, but we made intervalDecimal required to help simple importers to ignore the fractions mechanism.
* indeed, we should add: ... as defined in Table 5.

Comment (andreas-junghanns) [reply docs/examples/scheduled_execution.xml:21]: We need a new PR as this one was already merged. Will you (@MBlesken) provide one?

**Pull request merged**